Library/Formula/vegeta.rb

```diff
@@ -0,0 +1,28 @@
+require "language/go"
```

Comment:
I'd like users to not have to install `go` to install `vegeta`. For this purpose, I already provide [releases](https://github.com/tsenart/vegeta/releases). Could we use these instead please?